QAOAKit/build_tables.py

```diff
@@ -15,146 +15,179 @@
         # remove top folder
-        folder = list(Path(build_tables_folder, '../data/').glob('QAOAKit-data-*'))
-        assert(len(folder) == 1)
+        folder = list(Path(build_tables_folder, "../data/").glob("QAOAKit-data-*"))
+        assert len(folder) == 1
         folder = folder[0]
-        for subfolder in folder.glob('*'):
-            if subfolder.stem == '.gitignore':
+        for subfolder in folder.glob("*"):
+            if subfolder.stem == ".gitignore":
                 continue
             subfolder = subfolder.absolute()
             parent_dir = subfolder.parents[1]
             subfolder.rename(parent_dir / subfolder.name)
         shutil.rmtree(folder)
 
+
 def build_graph2angles():
     print(f"Building graph2angles table...")
-    
+
     tables = {}
-    
-    for n_qubits in tqdm(range(3,10)):
+
+    for n_qubits in tqdm(range(3, 10)):
         tables[n_qubits] = {}
-        
-        for p in tqdm(range(1,4), leave=False):
+
+        for p in tqdm(range(1, 4), leave=False):
             tables[n_qubits][p] = {}
-            
-            df = load_results_file_into_dataframe(n_qubits,p)
-    
+
+            df = load_results_file_into_dataframe(n_qubits, p)
+
             for index, data in df.iterrows():
-                if data['p'] != p:
-                    assert(np.isclose(data['C_{true opt}'], data['C_opt']))
+                if data["p"] != p:
+                    assert np.isclose(data["C_{true opt}"], data["C_opt"])
                 beta = [data[f"beta_{i}/pi"] for i in range(p)]
                 gamma = [data[f"gamma_{i}/pi"] for i in range(p)]
-                tables[n_qubits][p][int(index)] = {'beta':copy.deepcopy(beta), 'gamma':copy.deepcopy(gamma)} 
-    
-    pickle.dump(tables, open(Path(build_tables_folder, f"../data/lookup_tables/graph2angles.p"), "wb"))
+                tables[n_qubits][p][int(index)] = {
+                    "beta": copy.deepcopy(beta),
+                    "gamma": copy.deepcopy(gamma),
+                }
+
+    pickle.dump(
+        tables,
+        open(Path(build_tables_folder, f"../data/lookup_tables/graph2angles.p"), "wb"),
+    )
     del tables
     print(f"Done building graph2angles table.")
 
+
 def build_graph2pynauty():
     print(f"Building graph2pynauty table...")
-    
+
     tables = {}
-    
-    for n_qubits in tqdm(range(3,10)):
-    
-        with open(Path(build_tables_folder, "../data/qaoa-dataset-version1/Graphs/graph"+str(n_qubits)+"c.txt")) as f:
-            for graph in tqdm(range(1,n_graphs[n_qubits]+1), leave=False):
-                f.readline(-1)#first line is blank
-                line_with_id = f.readline(-1) #second line has graph number and order
-                graph_id, graph_order = [int(x) for x in re.split(' |, |. |.\n', line_with_id) if x.isdigit()]
-                assert(graph_order == n_qubits)
-                edges=[]
-                G=nx.Graph()
+
+    for n_qubits in tqdm(range(3, 10)):
+
+        with open(
```

Comment:
This is another change I'm not super happy about... I think it's harder to read. I'm not sure how hard it would be to configure the linter. In general, my feeling is that "making linter happy" is one of those things that feel useful, but are not actually useful.

Reply:
I think Black is an extremely opinionated linter. I don't feel strongly either way about this change, but with pre-commit hooks it shouldn't be something you have to think about. I'll defer to you on this though.

Reply:
I guess I can live with it. Let's see how annoying this is, can always turn it off if linter is too much work to deal with.